docpages/example_programs/interactions_and_components/making_threads.md

```diff
@@ -0,0 +1,115 @@
+\page making_threads Creating and talking in a thread
+
+A new feature added to Discord recently is `Threads`, these allow you to break off a message into a different "channel", without creating a whole new channel.
+
+In this tutorial, we'll be going through how to create a thread and how to talk in a thread.
+
+First, let's go through creating a thread.
+~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+
+int main()
+{
+    /* Create the bot */
+    dpp::cluster bot("token");
+
+    bot.on_log(dpp::utility::cout_logger());
+
+    /* The event is fired when the bot detects a message in any server and any channel it has access to. */
+    bot.on_slashcommand([&bot](const dpp::slashcommand_t& event) {
+		/* Check which command they ran */
+		if (event.command.get_command_name() == "create-thread") {
+			/* Here we create a thread in the current channel. It will expire after 60 minutes of inactivity. We'll also allow other mods to join, and we won't add a slowdown timer. */
+			bot.thread_create("Cool thread!", event.command.channel_id, 60, dpp::channel_type::CHANNEL_PUBLIC_THREAD, true, 0, [event](const dpp::confirmation_callback_t& callback) {
+				if (callback.is_error()) {
+					event.reply("Failed to create a thread!");
+					return;
+				}
+				
+				event.reply("Created a thread for you!");
+			});
+		}
+    });
+    
+    bot.on_ready([&bot](const dpp::ready_t& event) {
+		if (dpp::run_once<struct register_bot_commands>()) {
```

Comment:
Indentation moment

Reply:
The indentation appears to just be that line that are indented once, aren't using tab indents (and are also only at size 4). This has been rectified.